cloudferrylib/os/actions/check_config_quota_neutron.py

```diff
@@ -70,8 +70,14 @@ def get_tenants_without_quotas(self, tenants_src, list_quotas):
 
     def get_src_tenants(self, search_opts):
         identity_src = self.src_cloud.resources[utl.IDENTITY_RESOURCE]
-        identity_info = identity_src.read_info(**search_opts)
-        tenants = identity_info['tenants']
-        tenants_dict = {ten['tenant']['id']: ten['tenant']['name'] for ten in
-                        tenants}
+
+        if search_opts.get('tenant_id'):
+            filter_tenant_id = search_opts['tenant_id'][0]
```

Comment:
Let's don't make things worse and assume that we only have one tenant_id in filters. Please write this code like there are many tenant IDs in filters.